LoopKitUI/ViewModels/DisplayGlucoseUnitObservable.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DisplayGlucoseUnitObservable.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2021-03-10.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import Combine
+import HealthKit
+
+public class DisplayGlucoseUnitObservable: ObservableObject {
+    @Published public private(set) var displayGlucoseUnit: HKUnit {
+        didSet {
+            updatePublisher.send()
+        }
+    }
+
+    public let updatePublisher = PassthroughSubject<Void, Never>()
```

Comment:
Don't you get this by default because of the `@Published` and using `$displayGlucoseUnitObservable`? Perhaps I'm misunderstanding something about Combine.

Reply:
No... this is my misunderstanding. removed.